src/Promitor.Core.Scraping/Configuration/Model/Metrics/MetricDefinition.cs

```diff
@@ -17,6 +17,11 @@ public abstract class MetricDefinition
         /// </summary>
         public string Name { get; set; }
 
+        /// <summary>
+        ///     Override the Resource Group name at the metric level (to enable multiple RG)
```

Comment:
Let's try to be more descriptive here

```suggestion
        ///     Specify a resource group to scrape that defers from the default resource group.
        ///     This enables you to do multi-resource group scraping with one configuration file.
```